docs/release_notes/march_2024.md

```diff
@@ -0,0 +1,31 @@
+# February 2024
+## airbyte v0.51.0 to v0.56.0
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## ✨ Highlights
+
+Airbyte now supports **OpenID Connect (OIDC) SSO** for Airbyte Enterprise and Airbyte Cloud Teams. This enables companies to use Airbyte with Entra ID/AD via the OIDC protocol.
+
+Airbyte certified our [Microsoft SQL Server source](/integrations/sources/mssql) to support terabyte-sized tables, expanded datetime data types, and reliability improvements.
+
+Airbyte migrated our [Redshift destination](https://github.com/airbytehq/airbyte/pull/36255) to the [Destinations V2](./upgrading_to_destinations_v2) framework. This enables you to map tables one-to-one with your source, experience better error handling (particularly with large records), and deliver data incrementally. 
+
+
+## Platform Releases
+
+In addition to our OpenID Connect support, we also released: 
+
+- A major upgrade to our Docker and Helm deployments, which simplifies how external logs are configured. Learn more about the specific changes in our [migration guide](/deploying-airbyte/on-kubernetes-via-helm#migrate-from-old-chart-to-airbyte-v0520-and-latest-chart-version).
+
+
+- Our major version upgrades (Airbyte Cloud only) now only require manual upgrading when you are actively syncing a stream that has changed. Otherwise, syncs will continue as is and the version will be upgraded automatically for you.
+
+## Connector Improvements
+
+In addition to our MS-SQL certification, we also released a few notable Connector improvements:
+
+- We released several connector builder enhancements,including support for raw YAML blocks and the ability to reference custom python components to enable running connectors that use custom components. You can also modify the start date when testing and adjust page/slice/record limits.
```

Comment:
Hmm, I would love to document those better, but I don't have anything specific in mind to link to.

Nit: ability to reference custom components will only work in development / local Builder. 

Other builder improvements:
- Fixed bugs in page size and interpolation inputs: https://github.com/airbytehq/airbyte-platform-internal/pull/11926
- Switching from YAML to UI is faster now: https://github.com/airbytehq/airbyte-platform-internal/pull/11848
- Fixed some annoying layout Builder bugs: https://github.com/airbytehq/airbyte-platform-internal/pull/11695

Reply:
Added all three!